data/test/scenarios/events-test_filters.cfg

```diff
@@ -148,6 +148,157 @@
     [/event]
 ) (SIDE1_RECRUIT=Peasant,Woodsman)}
 
+{GENERIC_UNIT_TEST test_ability_id_active (
+    [event]
+        name=start
+        {VARIABLE triggers 0}
+        [unit]
+            id=alex
+            name=_"Alex"
+            x,y=12,4
+            type=Elvish Archer
+            side=1
+        [/unit]
+        [object]
```

Comment:
This needs to have a `[filter]` tag to match the unit it applies to.

You can probably just use `id=alice` or `id=bob`, no need to create a third unit, is there?